Makefile

```diff
@@ -10,7 +10,7 @@ DATA_PROTO_DIR=internal
 
 # Needed to be able to build amd64 binaries on MacOS M1
 DOCKER_DIR=test/docker
-DOCKER_COMPOSE=COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose -f ${DOCKER_DIR}/docker-compose.yml
+DOCKER_COMPOSE=COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ${DOCKER_DIR}/docker-compose.yml
```

Comment:
Should we add a version check for docker to ensure clients clearly see what's going on?

Reply:
Also, do we need to amed the `.github/workflows/go-test.yaml` GitHub workflow as well?

Reply:
I think Compose V2 should be included in most recent installations.

I'll fix go-test.yaml (Good catch, thanks!)